galaxy/nginx.conf

```diff
@@ -13,6 +13,9 @@ http {
     sendfile        on;
 
     keepalive_timeout  65;
+    types_hash_max_size 2048;
+    types_hash_bucket_size 64;
```

Comment:
The default value of types_hash_bucket_size is already 64 (http://nginx.org/en/docs/http/ngx_http_core_module.html#types_hash_bucket_size) can we remove this line?